### ISSUE TITLE
Rendering of @proceedings is broken

### ISSUE THREAD

**Issue by domhardt** [plk/biblatex-apa#119]
Here is the minimal example:
```
\documentclass{article}
\usepackage[utf8]{inputenc}
\usepackage[english]{babel}
\usepackage[babel=true,english=american]{csquotes} 
\usepackage[style = apa, backend = biber, isbn = true, doi = true]{biblatex}

\begin{filecontents*}{lit.bib}
@inproceedings{Ackermann2017,
	Author = {Ackermann, Laura and Mugge, Ruth and Schoormans, Jan PL},
	Crossref = {PLATE2017},
	Doi = {10.3233/978-1-61499-820-4-1},
	Pages = {1-4},
	Date = {2017},
	Title = {Consumers' attitudes towards product care: an exploratory study of motivators, ability factors and triggers}}

@proceedings{PLATE2017,
	Location = {Amsterdam, The Netherlands},
	Editor = {Bakker, Conny and Mugge, Ruth},
	Isbn = {978-1-61499-820-4},
	Publisher = {IOS Press},
	Title = {{PLATE}: Product Lifetimes And The Environment},
	Date = {2017}}
\end{filecontents*}
\addbibresource{lit.bib}

\begin{document}
\nocite{Ackermann2017}
\printbibliography 
\end{document}
```

Here is the result:
<img width="463" alt="Bildschirmfoto 2020-07-14 um 21 45 17" src="https://user-images.githubusercontent.com/1955290/87469271-50c92e80-c61b-11ea-9702-ecf508d163fe.png">
And here is the list of the problems:
1. Editors shouldn't be enclosed in parentheses, only the abbreviation Eds. should.
2. Title of the proceedings is missing.
3. Publisher of the proceedings is missing.
~~4. Location of the proceedings is missing.~~ updated 2020-07-15
~~5. ISBN of the proceedings is missing.~~ updated 2020-07-15

System information:
MacTeX-2020 incl. updates installed by TeX Live Utility
APA BibLATEX style Version 9.13
BibLATEX Version 3.14

UPDATE 2020-07-15
Problems 4 and 5 are no problems any more because location and ISBN is not required in APA 7.

**Comment by plk** [comment]
9.16 is about to be released if you can test it there and so we can see what the current state of this is.

**Comment by domhardt** [comment]
I just verified that the problems still existed with version 9.15 that was part of my MacTeX installation (MacTeC 2022, all updates installed). Please find the result of the example for version 9.15 below:
<img width="589" alt="Bildschirmfoto 2022-06-22 um 20 08 36" src="https://user-images.githubusercontent.com/1955290/175107214-df9408b5-7af8-4647-8417-c91683b2b0d0.png">

**Comment by domhardt** [comment]
Using the new version 9.16 installed locally (in ~/Library/texmf/tex/latex/biblatex-apa) breaks the ability to compile the tex file. Please find the log file attached.
![Bildschirmfoto 2022-06-22 um 20 42 46](https://user-images.githubusercontent.com/1955290/175112691-27baf414-3d45-420d-9384-ff3b5fa622d5.png)

[test.log](https://github.com/plk/biblatex-apa/files/8960512/test.log)

**Comment by plk** [comment]
That 9.16 error is expected - as per the doc - you need bibaltex 3.18 and biber 2.18 which haven't been fully released yet - it's in progress. I won't push 9.16 to CTAN and TL until biblatex and biber released are done.

**Comment by plk** [comment]
With 915, in the file `apa.bbx`, after the line:

```
\step[notfield=editor, final]
```

can you insert the line:

```
\step[notfield=crossref, final]
```

and try it again?

**Comment by domhardt** [comment]
> That 9.16 error is expected - as per the doc - you need bibaltex 3.18 and biber 2.18 which haven't been fully released yet - it's in progress. I won't push 9.16 to CTAN and TL until biblatex and biber released are done.

Oh, sorry. That was my fault. I'm about to test your other suggestion.

**Comment by domhardt** [comment]
> With 915, in the file `apa.bbx`, after the line:
> 
> ```
> \step[notfield=editor, final]
> ```
> 
> can you insert the line:
> 
> ```
> \step[notfield=crossref, final]
> ```
> 
> and try it again?

As I found no way to locally install the latest unreleased biber version I did that for the version 9.15 (was line 369 there). The result looks very promising! Problems 1-3 seem to be fixed. Great work!

<img width="577" alt="Bildschirmfoto 2022-06-22 um 21 08 29" src="https://user-images.githubusercontent.com/1955290/175117506-3a933373-7fa6-4699-bf02-3cad2ce17651.png">

**Comment by plk** [comment]
Thanks for confirming - this fix will be in 9.16 when released in the next week or so.

**Comment by domhardt** [comment]
> Thanks for confirming - this fix will be in 9.16 when released in the next week or so.

Perfect! I can't wait to see all updates (biblatex, biber, apa) coming in over the regular update procedure and working together. Keep up the good work!
